keyboards/suavity/ehan/info.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
This file's... not ideal... but it'll be easier to fix it once the PR is merged.

![image](https://user-images.githubusercontent.com/18669334/179303359-46ee1532-9d0c-492d-95bd-9b2560a2811c.png)